ci_framework/roles/use_install_yamls/molecule/default/prepare.yml

```diff
@@ -0,0 +1,30 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Prepare
+  hosts: all
+  vars:
+    cifmw_basedir: "/tmp/ci-framework"
+  roles:
+    - role: test_deps
+    - role: ci_setup
+  tasks:
+    - name: Fetch install_yamls repository
```

Comment:
this isn't needed anymore, it's fetched in the zuul prep steps already - located in {{ansible_user_dir}}/src/github.com/openstack-k8s-operators/install_yamls (check the other molecule tests refering it)